v2/stack_analyses.yaml

```diff
@@ -557,6 +564,21 @@ components:
           registered: '#/components/schemas/StackAnalysesResultForRegisteredUser'
           freetier: '#/components/schemas/StackAnalysesResultForFreeTier'
       properties:
+        version:
+          type: string
+          example: v2
+        started_at:
```

Comment:
What is purpose of the started and ended time? Is it displayed in UI?

Reply:
yep., there is no real need right time. @dgpatelgit , Can you elaborate why you need this?

Reply:
Removed it